### PR TITLE
Chore: Better form validation in 'Create Chat' modal

### PR DESCRIPTION
**Pull request by stas-kh**
**Screenshots**:

Screenshot 2023-02-20 at 13 31 09
Screenshot 2023-02-20 at 13 31 17

### CODE REVIEW

**Review by lazynina — COMMENTED** [review]
What happens when I try to add a user who hasn't registered the default key?

**Comment by lazynina** [comment]
looks like we still get the toast notification when we try to add a user who hasn't registered the default key. Do you have any ideas on how we can use a similar approach to what you currently have to show an error when such a user is selected?

**Comment by stas-kh** [comment]
Hey @lazynina 

For now, I decided to forbid closing "Start-group-chat" and "Manage-members" dialogs when clicking outside. But I wanted to keep closing the dialog when the `Esc` button was pressed, so I added a custom hook.


https://user-images.githubusercontent.com/10476834/222763215-0088e6c2-0ebd-4075-a476-365803848a3a.mov

File: src/components/form/my-error-label.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
```
Comment (lazynina) [inline src/components/form/my-error-label.tsx]: Do we need to import React?

File: src/components/form/my-input.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
```
Comment (lazynina) [inline src/components/form/my-input.tsx]: Do we need this import?

**Review by lazynina — APPROVED** [review]

**Pull request merged**